### PR TITLE
Pull a few utilities up from individual apps.

### PR DESCRIPTION
**Pull request by z00b**

### CODE REVIEW

File: src/cljs/parseapp_cljs/express.cljs [hunk 1]
```diff
@@ -21,3 +22,32 @@
...
+(defn force-login
+  ([url-getter]
+    (fn [handler]
+      (fn [request response]
+        (if (logged-in?)
+          (handler request response)
+          (if (.-xhr request)
+            (render-json response 400 {:error "You must be logged in to perform this action."})
+            (redirect-after-post response (url-getter request)))))))
+  ([] (force-login #("/"))))
```
Comment (travis) [inline src/cljs/parseapp_cljs/express.cljs]: won't #("/") try to run the string as a function? I think you need to use `constantly`

**Comment by travis** [comment]
:cake: otherwise

File: src/cljs/parseapp_cljs/express.cljs (hunk 1 above)
Comment (z00b) [inline src/cljs/parseapp_cljs/express.cljs]: i thought about that.  then i thought i tested it.  now i question my test.

Comment (travis) [inline src/cljs/parseapp_cljs/express.cljs]: ever figure out what was going on here?

**Comment by z00b** [comment]
finally updated.  fixed `force-login`, fixed compile warnings on `URL` and removed calls to `timeout`

**Comment by travis** [comment]
:cake:

**Pull request merged**